sunpy/net/tests/test_vso.py

```diff
@@ -438,8 +438,7 @@ def test_vso_hmi(client, tmpdir):
     # Extract the DRIs from the request
     dris = dr.request.datacontainer.datarequestitem
 
-    # 3 HMI series and one AIA
-    assert len(dris) == 4
+    assert len(dris) == 90
```

Comment:
I'm just trying to understand this.
What causes this increase in the number of results?
Further, you should update the comment which still mentions 4 results.

Reply:
Thanks, I need to remove that line.
The number of results are still same, but what happened is earlier there were multiple files in same requestitem, so number of results are less.

Now all fileids have different `DataRequestitem`, so no. of results increased. But the files shall always be same.

Reply:
Cool!